sdk/scheduler/src/test/resources/test-update-pod-a.yml

```diff
@@ -0,0 +1,36 @@
+name: "test"
```

Comment:
Alternatively, if not that, then maybe a Builder class for `RawSpec` that handles the generation of sensible defaults for unspecified fields? I reaaaally don't want to end up with a folder full of literally 100s of test YAMLs, which I *promise* will be the end result here, and which will be a real maintainability burden should we end up adding or modifying features that bubble up to YAML.